frontend/src/config/i18n/@types/i18next.d.ts

```diff
@@ -0,0 +1,8 @@
+import { resources, defaultNS } from '..';
+
+declare module 'i18next' {
+  interface CustomTypeOptions {
+    defaultNS: typeof defaultNS;
+    resources: typeof resources['en'];
```

Comment:
de?

Reply:
It's hard to dynamically type based on the chosen language.
We will have to define a base language that defines all the keys for typing and derived languages like `de`.